gems/sorbet-runtime/lib/types/enum.rb

```diff
@@ -47,7 +47,7 @@ class T::Enum
   SerializedVal = T.type_alias {T.untyped}
   private_constant :SerializedVal
 
-  ## Enum class methods ##
+  ### Enum class methods ###
```

Comment:
The double `#` seemed to have some [special meaning](https://github.com/lsegal/yard/blob/main/lib/yard/parser/ruby/legacy/statement_list.rb#L256) to `yard`, leading to a warning 🤷